pytest_invenio/fixtures.py

```diff
@@ -589,3 +591,70 @@ def create_bucket_from_dir(source_dir, location_obj=None):
         db.session.commit()
         return bucket_obj
     return create_bucket_from_dir
+
+
+@pytest.fixture(scope='module')
+def inv_cache(appctx):
+    """Setup Invenio Cache fixture.
+
+    Scope: Function
+
+    This fixture creates an invenio-cache instance that uses Redis and pushes
+    all
+    """
+    from invenio_cache import current_cache
+    return current_cache
+
+
+@pytest.fixture(scope='function')
+def inv_cache_clear(inv_cache, app_config):
+    """Setup Invenio cache fixture and clear keys after use.
+
+    Scope: Function
+
+    This fixture creates an invenio-cache instance and clears all keys
+    that are managed by the cache instance that start with CACHE_KEY_PREFIX.
+    By default this fixture uses the `cache::` prefix, to set a custom prefix
+    the following to your test file:
+
+    .. code-block:: python
+
+        @pytest.fixture(scope='module')
+        def app_config(app_config):
+            app_config['CACHE_KEY_PREFIX'] = 'custom_key'
+            return app_config
+    """
+    yield inv_cache
+
+    from redis import Redis
+    redis_client = Redis()
+
+    print(app_config)
+
+    prefix = app_config.get('CACHE_KEY_PREFIX', 'cache::')
```

Comment:
Can this be done via https://flask-caching.readthedocs.io/en/latest/api.html#flask_caching.Cache.clear

Reply:
Yes, thanks for pointing this out.